homeassistant/components/home_connect/strings.json

```
@@ -135,43 +135,231 @@
+        "name": "Freezer door alarm",
         "state": {
           "confirmed": "[%key:component::home_connect::common::confirmed%]",
           "present": "[%key:component::home_connect::common::present%]"
         }
       },
-      "alarm_sensor_freezer": {
+      "refrigerator_door_alarm": {
+        "name": "Refrigerator door alarm",
         "state": {
+          "off": "[%key:common::state::off%]",
           "confirmed": "[%key:component::home_connect::common::confirmed%]",
           "present": "[%key:component::home_connect::common::present%]"
         }
       },
-      "alarm_sensor_temp": {
+      "freezer_temperature_alarm": {
+        "name": "Freezer temperature alarm",
         "state": {
+          "off": "[%key:common::state::off%]",
           "confirmed": "[%key:component::home_connect::common::confirmed%]",
           "present": "[%key:component::home_connect::common::present%]"
         }
       },
-      "alarm_sensor_coffee_bean_container": {
+      "bean_container_empty": {
+        "name": "Bean container empty",
         "state": {
+          "off": "[%key:common::state::off%]",
           "confirmed": "[%key:component::home_connect::common::confirmed%]",
           "present": "[%key:component::home_connect::common::present%]"
         }
       },
-      "alarm_sensor_coffee_water_tank": {
+      "water_tank_empty": {
+        "name": "Water tank empty",
         "state": {
+          "off": "[%key:common::state::off%]",
           "confirmed": "[%key:component::home_connect::common::confirmed%]",
           "present": "[%key:component::home_connect::common::present%]"
         }
       },
-      "alarm_sensor_coffee_drip_tray": {
+      "drip_tray_full": {
+        "name": "Drip tray full",
         "state": {
+          "off": "[%key:common::state::off%]",
           "confirmed": "[%key:component::home_connect::common::confirmed%]",
           "present": "[%key:component::home_connect::common::present%]"
         }
       }
+    },
+    "switch": {
+      "power": {
+        "name": "Power"
+      },
+      "child_lock": {
+        "name": "Child lock"
+      },
+      "cup_warmer": {
+        "name": "Cup warmer"
+      },
+      "refrigerator_super_mode": {
+        "name": "Refrigerator super mode"
+      },
+      "freezer_super_mode": {
+        "name": "Freezer super mode"
+      },
+      "eco_mode": {
+        "name": "Eco mode"
+      },
+      "sabbath_mode": {
+        "name": "Sabbath mode"
+      },
+      "vacation_mode": {
+        "name": "Vacation mode"
+      },
+      "fresh_mode": {
+        "name": "Fresh mode"
+      },
+      "dispenser_enabled": {
+        "name": "Dispenser",
+        "state": {
+          "off": "[%key:common::state::disabled%]",
+          "on": "[%key:common::state::enabled%]"
+        }
+      },
+      "door_assistant_fridge": {
+        "name": "Fridge Door assistant"
+      },
+      "door_assistant_freezer": {
+        "name": "Freezer Door assistant"
```

```suggestion
        "name": "Fridge door assistant"
      },
      "door_assistant_freezer": {
        "name": "Freezer door assistant"
```
What even does these mean?

---

From the API:
> Enables/disables the Automatic Door Opening Refrigerator / Freezer Compartment